src/ildasm/dis.cpp

```diff
@@ -315,7 +315,6 @@ BOOL enumEHInfo(const COR_ILMETHOD_SECT_EH* eh, IMDInternalImport *pImport, DWOR
     COR_ILMETHOD_SECT_EH_CLAUSE_FAT ehBuff;
     const COR_ILMETHOD_SECT_EH_CLAUSE_FAT* ehInfo;
     BOOL    fTryInCode = FALSE;
-    if(g_ehInfo) VDELETE(g_ehInfo);
```

Comment:
My understanding is "g_ehInfo" is initialized to NULL. If it is not allocated before (for the first entrance), then we don't delete this. How come we deleted this without allocation?